dbadmin/migrations/021_cves_sorting_names.up.sql

```diff
@@ -0,0 +1,2 @@
+CREATE COLLATION IF NOT EXISTS numeric (provider = icu, locale = 'en-u-kn-true');
```

Comment:
It's not needed to create the collation here, because it exists since version 013.